Tema 4/formulare modificat/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Formular</title>
+        <link rel="stylesheet" href="style.css"
+    </head>
+    <body class="body">
+        <form class="form">
+            <fieldset style="display: inline-block;"><legend>Pizza Order</legend>
+            <label for="your-name">Your name:</label>
```

Comment:
un label este legat de input prin id
Astfel, la inputul de mai jos trebuia:

```suggestion
           <input type="text" id="your-name" name="your-name" value="Malina" minlength="3" maxlength="10" required placeholder="Your name"/>
           
```